src/App.jsx

```diff
@@ -73,15 +79,18 @@ const App = () => {
      * @function fetchData
      */
     async function fetchData() {
-      await generateUsersList(session);
-      await generateActivityTTL(session);
```

Comment:
This function is needed for generating/updating the active.ttl used for Last Active Date. Should be kept.

Reply:
This functionality has been moved to `updateUserActivity` in `model-helpers/User.js`